language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/AbstractCompletionProvider.java

```diff
@@ -270,7 +271,8 @@ private List<LSCompletionItem> getTypeItems(BallerinaCompletionContext context)
                 new SnippetCompletionItem(context, Snippet.KW_DISTINCT.get()),
                 new SnippetCompletionItem(context, Snippet.DEF_OBJECT_TYPE_DESC_SNIPPET.get()),
                 new SnippetCompletionItem(context, Snippet.KW_TRUE.get()),
-                new SnippetCompletionItem(context, Snippet.KW_FALSE.get())
+                new SnippetCompletionItem(context, Snippet.KW_FALSE.get()),
+                new SnippetCompletionItem(context, Snippet.KW_NIL.get())
```

Comment:
Is there a specific reason for not adding `annotation` keyword as a type here instead of adding it in the constant declaration node context

Reply:
Assume someone wants to declare an annotation but he does not know that it must be a constant. Shouldn't we provide `annotation` keyword as a type as a forward completion item

Reply:
An annotation can be defined only in the module part context. We are already suggesting the annotation definition snippet there. So, is there any use case of suggesting annotation keyword there again?

Adding annotation keyword as a type will make it available in other non-relevant contexts as well.

Reply:
Let's not add it then. +1